README.md

```diff
@@ -1,17 +1,28 @@
-# Socket Component
+# react/socket
```

Comment:
Agreeing with @jsor here, the old `Socket Component` looks better. Some random other ideas for it:

- `Client and Server Socket Component`
- `Socket Component (Client and Server)`
- `Component providing Client and Server`
- `Client/Server Socket Component`